src/screens/HomePage/HomePage.tsx

```diff
@@ -145,7 +145,7 @@ export default function HomePage() {
             </ColumnCentered>
 
             <MapBlock
-              title="Community levels"
+              title="COVID Community Level"
```

Comment:
It looks like community and level should be lowercase?

Reply:
Yup, should be all lowercase actually. Thanks Michael!